### ISSUE TITLE
May I show some part of the next slide in react-material-ui-carousel?

### ISSUE THREAD

**Issue by grit-ki** [Learus/react-material-ui-carousel#164]
Hi. I am intending to show some part of the next slide with this lib. As you can see, some part of the next slide is visible on the first slide. Is it feasible? I've looked through all documents but couldn't make it..

I would really appreciate if you give me some tips.

<img width="327" alt="1111" src="https://user-images.githubusercontent.com/89117833/158777089-6b38f80f-6518-4a2a-8951-ba884aefc158.png">

**Comment by Learus** [comment]
This is an interesting idea. I think it should be feasible if you mess around with the values of the framer/motion animations.
Check the `variants` variable in the `CarouselItem`. [Link for easy lookup](https://github.com/Learus/react-material-ui-carousel/blob/master/src/components/CarouselItem.tsx#L62)

I'm planning on creating some more animations and other stuff, so this could be an idea that will be brought to existence when that happens

**Comment by averybross** [comment]
Hey! Any update on this?

**Comment by Learus** [comment]
@averybross No, not yet. I haven't had the time to work on the library at all, in the past months. Stay tuned tho! 😄
